.github/workflows/test-chart.yaml

```diff
@@ -24,6 +24,16 @@ on:
   workflow_dispatch:
 
 jobs:
+  lint_shell_scripts:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.3
+        name: Run shellcheck linter
+        with:
+          extra_args: --config .pre-commit-config-shellcheck.yaml
```

Comment:
```suggestion
      - name: Run shellcheck linter
        run: pre-commit run -a --config .pre-commit-config-shellcheck.yaml
```
It's still not checking anything though 😐
![image](https://user-images.githubusercontent.com/1644105/135858251-2fc64e68-9a6a-4b26-816b-e86ab4fa0b64.png)

Could try running pre-commit explicitly?
```sh
$ pre-commit run -a --config .pre-commit-config-shellcheck.yaml
[INFO] Initializing environment for https://github.com/gruntwork-io/pre-commit.
Shellcheck Bash Linter...................................................Passed
```